packages/components/src/FilterSelect/FilterSelect.spec.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react"
+import { render, screen, waitFor } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { FilterButton } from "~components/FilterButton"
+import { FilterSelect, FilterSelectProps } from "./FilterSelect"
+import { mixedMockItemsUnordered } from "./docs/mockData"
+
+const FilterSelectWrapper = ({
+  isOpen: propsIsOpen = false,
+  items = mixedMockItemsUnordered,
+  selectedKey,
+  onSelectionChange,
+  ...restProps
+}: Partial<FilterSelectProps>): JSX.Element => {
+  const [isOpen, setIsOpen] = useState<boolean>(propsIsOpen)
+  const [selected, setSelected] = React.useState<
+    FilterSelectProps["selectedKey"]
+  >(selectedKey ?? null)
+
+  return (
+    <FilterSelect
+      label="Coffee"
+      isOpen={isOpen}
+      setIsOpen={setIsOpen}
+      renderTrigger={(triggerButtonProps): JSX.Element => (
+        <FilterButton {...triggerButtonProps} />
+      )}
+      items={items}
+      selectedKey={selected}
+      onSelectionChange={(key): void => {
+        setSelected(key)
+        onSelectionChange?.(key)
+      }}
+      {...restProps}
+    />
+  )
+}
+
+describe("<Select>", () => {
+  it("does not show the options initially", () => {
+    render(<FilterSelectWrapper />)
+    expect(screen.queryByRole("listbox")).not.toBeInTheDocument()
+  })
+
+  it("shows the options initially when isOpen is true", async () => {
+    render(<FilterSelectWrapper isOpen />)
+    await waitFor(() => {
+      expect(screen.queryByRole("listbox")).toBeVisible()
+    })
+  })
+
+  describe("Trigger", () => {
+    it("shows the selected option in the button", () => {
```

Comment:
Is it necessary to test this since it's tested in `Filter`? ie. if it fails in `Filter` then we can assume it failed here too.

Reply:
In a normal abstraction, we absolutely do not need to test it. But of course, react-aria is so opinionated and does funny stuff in its state that I can't ensure won't change, I figured I'd add a test.

That being said, I realise I have an example of a selected option in the Sticker Sheet, so I suppose we could probably get away without this unit test?

Reply:
Hm I think keep it, probably a tighter feedback loop compared to waiting on Chromatic